### PR TITLE
Changed forgot password handling and Made image and title clickabe while viewing blogs. Also fixed the super annoying Log out button so now it is in position with the Contact:) Additional fix for title in blogs.

### PR DESCRIPTION
**Pull request by Sanyi0001**
Instead of displaying the "If this email address exists in our system, we have emailed you a password reset link!" text with red, as an error with the "whoops" text, it will be displayed with green as a success without the error message. The user won't be able to determine whether the requested email address is in the database or not.

### CODE REVIEW

**Comment by Sanyi0001** [comment]
"Made image and title clickable, link points to an external site defined by link while creating a blog. Fix
: added space between blog activity name and blog title.

**Review by koenhamstra — APPROVED** [review]
Looks good and works fine.

**Review by avanheeren — APPROVED** [review]

**Comment by AlexGit123** [comment]
Once conflicts are fixed, the PR can be merged.

**Comment by Sanyi0001** [comment]
Conflict solved

File: resources/views/blogs/create.blade.php [hunk 1]
```diff
@@ -23,7 +23,7 @@
                                 @foreach ($programs as $program)
                                     <option
                                         value="{{$program->id}}" {{(old('program')==$program->id ? "selected":"")}}>{{$program->name}}
-                                    </option>
+                                        </option>
```
Comment (Koreyoshi159) [inline resources/views/blogs/create.blade.php]: spaces ??

File: resources/views/sdg.blade.php [hunk 2]
```diff
@@ -27,18 +27,21 @@
     <div class="columns columns-container is-multiline">
 
         @foreach($sdg->blogs as $blog)
+
```
Comment (Koreyoshi159) [inline resources/views/sdg.blade.php]: what is this line for ?

File: resources/views/sdg.blade.php [hunk 3]
```diff
@@ -27,18 +27,21 @@
...
             @continue($blog->visibility == 0)
             <div class="column is-half">
                 <div class="card">
                     <div class="card-image">
                         <figure class="image is-2by1">
-                            <img src="https://bulma.io/images/placeholders/1280x960.png" alt="Placeholder image">
+                            <a href="//{{ $blog->link }}"><img src="https://bulma.io/images/placeholders/1280x960.png"
```
Comment (Koreyoshi159) [inline resources/views/sdg.blade.php]: Just put the image to a new line instead of break the img

File: resources/views/sdg.blade.php [hunk 4]
```diff
@@ -27,18 +27,21 @@
...
+                                                               alt="Placeholder image"></a>
                         </figure>
                     </div>
                     <div class="card-stacked">
                         <div class="card-content">
                             <div class="media-content">
-                                <p class="title is-4">{{ $blog->activity->name }} <i>Blog title</i></p>
+                                <a class="title is-4" href="//{{ $blog->link }}">{{ $blog->activity->name }}<i> Blog
```
Comment (Koreyoshi159) [inline resources/views/sdg.blade.php]: `<a class="title is-4" href="//{{ $blog->link }}"> put enter here
    {{ $blog->activity->name }}<i>Blogtitle</i> and here
</a>`
Do something like this

File: resources/views/sdg.blade.php [hunk 5]
```diff
@@ -48,12 +51,12 @@
                                 Subgoal: {{ $blog->subSdgs }}<br>
                                 Publisher: {{ $blog->contact_name }} <br>
                                 Updated at: {{ $blog->updated_at }} <br>
+
```
Comment (Koreyoshi159) [inline resources/views/sdg.blade.php]: this line ?

File: resources/views/sdg.blade.php [hunk 6]
```diff
@@ -48,12 +51,12 @@
...
                             </div>
                         </div>
                     </div>
                 </div>
-            </div>
```
Comment (Koreyoshi159) [inline resources/views/sdg.blade.php:55]: why is this div removed ?

File: resources/views/sdg.blade.php [hunk 7]
```diff
@@ -48,12 +51,12 @@
...
-        @endforeach
+                @endforeach
 
-    </div>
+            </div>
```
Comment (Koreyoshi159) [inline resources/views/sdg.blade.php:59]: spaces ?

File: resources/views/sdg.blade.php [hunk 8]
```diff
@@ -25,20 +25,21 @@
...
-
         @foreach($sdg->blogs as $blog)
             @continue($blog->visibility == 0)
             <div class="column is-half">
                 <div class="card">
                     <div class="card-image">
                         <figure class="image is-2by1">
-                            <img src="https://bulma.io/images/placeholders/1280x960.png" alt="Placeholder image">
+                            <a href="//{{ $blog->link }}">
+                                <img src="https://bulma.io/images/placeholders/1280x960.png" alt="Placeholder image"></a>
```
Comment (Koreyoshi159) [inline resources/views/sdg.blade.php]: Put the a to the next line

File: resources/views/sdg.blade.php [hunk 9]
```diff
@@ -25,20 +25,21 @@
...
                         </figure>
                     </div>
                     <div class="card-stacked">
                         <div class="card-content">
                             <div class="media-content">
-                                <p class="title is-4">{{ $blog->activity->name }} <i>Blog title</i></p>
+                                <a class="title is-4" href="//{{ $blog->link }}">{{ $blog->activity->name }}
+                                    <i> Blog title</i></a>
```
Comment (Koreyoshi159) [inline resources/views/sdg.blade.php]: put the a on next line

File: resources/views/sdg.blade.php (hunk 8 above)
Comment (Sanyi0001) [reply resources/views/sdg.blade.php]: ?

Comment (Sanyi0001) [reply resources/views/sdg.blade.php]: Why?

**Pull request merged**